src/main/kotlin/com/example/api/Service/BookingService.kt

```diff
@@ -42,13 +42,33 @@ import javax.persistence.*
     }
 
 
-             fun deleteBooking(customerId: Int,id: Int): Optional<Booking> {
+             fun deleteBooking(customerId: Int,id: Int) {
                 val customer= customerRepository.findById(customerId)
                 if(customer.isEmpty){
                     throw EntityNotFoundException("customer not found.")
                 }
-               return bookingRepository.deleteById(customerId,id)
+                bookingRepository.deleteById(id)
```

Comment:
What happens when there is no booking with id?

Reply:
Added that in the new branch.